lua/telescope/pickers.lua

```diff
@@ -537,6 +537,9 @@ function Picker:find()
 
   self.original_win_id = a.nvim_get_current_win()
   _, self.original_cword = pcall(vim.fn.expand, "<cword>")
+  _, self.original_cWORD = pcall(vim.fn.expand, "<cWORD>")
+  _, self.original_cfile = pcall(vim.fn.expand, "<cfile>")
+  _, self.original_cline = pcall(vim.trim, vim.api.nvim_get_current_line())
```

Comment:
We are currently pcalling vim.trim, wouldnt it make more sense to pcall nvim_get_current_line

Reply:
fixed